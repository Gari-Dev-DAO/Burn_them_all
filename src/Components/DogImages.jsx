

const DogImages = () => {
  return (
    <div style={{display:'flex',flexWrap:'wrap',justifyContent:'center'}}>
     <div className='dogImgContainer'>
     <img src={require('../assests/Images/dogs.png')} alt='dog-seat' className='dogImgs'/>
     </div>
    <button className='nftbtn'>NFT Launching Soon</button>
    </div>
  )
}

export default DogImages