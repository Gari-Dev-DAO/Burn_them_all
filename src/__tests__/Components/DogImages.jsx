import { render } from "@testing-library/react";
import DogImages from "../../Components/DogImages";

describe("DogImages", () => {
  it("renders component", () => {
    const { container } = render(<DogImages />);
    expect(container).toMatchSnapshot();
  });
});
