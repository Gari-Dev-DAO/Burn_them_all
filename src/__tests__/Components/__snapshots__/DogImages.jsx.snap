// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DogImages renders component 1`] = `
<div>
  <div
    style="display: flex; flex-wrap: wrap; justify-content: center; align-items: center; width: 100%;"
  >
    <div
      class="dogImgContainer"
    >
      <img
        alt="dog-seat"
        class="dogImgs"
        src="dogs.png"
      />
    </div>
    <button
      class="nftbtn"
    >
      NFT Launching Soon
    </button>
  </div>
</div>
`;
